test/blackbox-tests/test-cases/too-many-parens/run.t

```diff
@@ -25,8 +25,19 @@ are readable.
   These parentheses are no longer necessary with dune, please remove them.
   [1]
 
-Checking that extra long stanzas (over 10 lines) are not printed
+Checking that extra long stanzas (over 10 lines) are truncated in the middle
   $ dune build --root d
   File "dune", line 3, characters 13-192:
+  3 |   (libraries (a
+  4 |               b
+  5 |               c
+  6 |               d
+  
+  ------
```

Comment:
That's probably a bit too large, can you align it to just before the position of `|` as in your comment before? I'm also curious about using `...` since that's more associated with cutting parts out (and that's what bucklescript uses).

Reply:
Good point. I've updated to use `...` That indeed looks nicer. Also they should now be aligned with the `|`